src/libponyrt/sched/mpmcq.c

```diff
@@ -40,7 +40,7 @@ void ponyint_mpmcq_init(mpmcq_t* q)
 
   atomic_store_explicit(&q->head, node, memory_order_relaxed);
 #ifdef PLATFORM_IS_X86
-  PONY_ABA_PROTECTED(mpmcq_node_t*) tail;
+  PONY_ABA_PROTECTED_PTR(mpmcq_node_t) tail;
```

Comment:
No need for `bigatomic_store_explicit` here. There is no concurrency issue when initialising the queue.